### PR TITLE
feat(storage.cold-archive): update the doc links for multi-language support

### PR DESCRIPTION
**Pull request by brofik**
ref: MANAGER-10257
Signed-off-by: Rafik Adiche <rafik.adiche.ext@corp.ovh.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-9069` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-10257 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
-  "pci_projects_project_storages_cold_archives_guides_billing_title": "Comprendre la facturation",
-  "pci_projects_project_storages_cold_archives_guides_billing_description": "Découvrez les règles de facturation",
-  "pci_projects_project_storages_cold_archives_guides_monitoring_title": "Monitorer ses archives",
-  "pci_projects_project_storages_cold_archives_guides_monitoring_description": "Comprendre les états des archives et les monitorer",
+  "pci_projects_project_storages_cold_archives_guides_overview_title": "Cold Archive - Overview",
+  "pci_projects_project_storages_cold_archives_guides_overview_description": "Discover the service, understand the capabilities and billing",
+  "pci_projects_project_storages_cold_archives_guides_first_step_title": "Cold Archive - First step",
+  "pci_projects_project_storages_cold_archives_guides_first_step_description": "Learn how to create buckets, archive, retrieve data, list metadata",
+  "pci_projects_project_storages_cold_archives_guides_manage_data_title": "Cold Archive - Manage your data",
+  "pci_projects_project_storages_cold_archives_guides_manage_data_description": "Upload and download data by API, CLI, or rCLone",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/translations/Messages_fr_FR.json:12]: Why using only English version ?

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archives.constants.js [hunk 2]
```diff
@@ -29,17 +29,31 @@ export const CHECK_PRICES_DOC_LINK = {
...
-    id: 'cold-archive-getting-started',
-    link:
-      'https://docs.ovh.com/fr/storage/s3/getting-started-with-cold-archive/',
+    id: 'overview',
+    links: {
+      default:
+        'https://docs.ovh.com/en/storage/object-storage/cold-archive/overview/',
+      fr:
+        'https://docs.ovh.com/fr/storage/object-storage/cold-archive/overview/',
+    },
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archives.constants.js:38]: i guess it's a good practice to write them in capital to indicate they are fixed values (constant).
```suggestion
    LINKS: {
      DEFAULT:
        'https://docs.ovh.com/en/storage/object-storage/cold-archive/overview/',
      FR:
        'https://docs.ovh.com/fr/storage/object-storage/cold-archive/overview/',
    },
```

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archives.constants.js [hunk 3]
```diff
@@ -29,17 +29,31 @@ export const CHECK_PRICES_DOC_LINK = {
...
   {
-    id: 'billing',
-    link: 'https://docs.ovh.com/gb/en/storage/s3/getting-started-with-s3/',
+    id: 'first_step',
+    links: {
+      default:
+        'https://docs.ovh.com/en/storage/object-storage/cold-archive/getting-started',
+      fr:
+        'https://docs.ovh.com/fr/storage/object-storage/cold-archive/getting-started',
+    },
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archives.constants.js:47]: Related to:
https://github.com/ovh/manager/pull/8553/files#r1046840238

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archives.constants.js [hunk 4]
```diff
@@ -29,17 +29,31 @@ export const CHECK_PRICES_DOC_LINK = {
...
   {
-    id: 'monitoring',
-    link: 'https://docs.ovh.com/fr/storage/s3/limitations/',
+    id: 'manage_data',
+    links: {
+      default:
+        'https://docs.ovh.com/en/storage/object-storage/cold-archive/manage-your-data',
+      fr:
+        'https://docs.ovh.com/fr/storage/object-storage/cold-archive/manage-your-data',
+    },
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archives.constants.js:56]: Related to:
https://github.com/ovh/manager/pull/8553/files#r1046840238

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/onboarding/onboarding.controller.js [hunk 5]
```diff
@@ -23,6 +24,9 @@ export default class PciStorageColdArchivesOnboardingController {
           description: this.$translate.instant(
             `pci_projects_project_storages_cold_archives_guides_${guide.id}_description`,
           ),
+          link:
+            guide.links[this.coreConfig.getUserLanguage()] ||
+            guide.links.default,
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/onboarding/onboarding.controller.js:29]:
```suggestion
            guide.LINKS[this.coreConfig.getUserLanguage()] ||
            guide.LINKS.DEFAULT,
```

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js [hunk 6]
```diff
@@ -24,7 +25,26 @@ export default /* @ngInject */ ($stateProvider) => {
               },
         ),
     resolve: {
-      guides: () => GUIDES,
+      guides: /* @ngInject */ (coreConfig, $translate) =>
+        reduce(
+          GUIDES,
```
Comment (y4nnL) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js:30]: I think we could do without importing `reduce` here
```suggestion
        GUIDES.reduce(
```

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js [hunk 7]
```diff
@@ -24,7 +25,26 @@ export default /* @ngInject */ ($stateProvider) => {
...
+            {
+              ...guide,
+              title: $translate.instant(
+                `pci_projects_project_storages_cold_archives_guides_${guide.id}_title`,
+              ),
+              description: $translate.instant(
+                `pci_projects_project_storages_cold_archives_guides_${guide.id}_description`,
+              ),
+              link:
+                guide.links[coreConfig.getUserLanguage()] ||
```
Comment (y4nnL) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js:42]: Are we sure about this ?
Don't we usually use the `ovhSubsidiary` ?
```suggestion
                guide.links[coreConfig.getUser().ovhSubsidiary] ||
```

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/onboarding/onboarding.controller.js [hunk 8]
```diff
@@ -23,6 +24,9 @@ export default class PciStorageColdArchivesOnboardingController {
           description: this.$translate.instant(
             `pci_projects_project_storages_cold_archives_guides_${guide.id}_description`,
           ),
+          link:
+            guide.links[this.coreConfig.getUserLanguage()] ||
```
Comment (y4nnL) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/onboarding/onboarding.controller.js:28]: Same as https://github.com/ovh/manager/pull/8553#discussion_r1048139625

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/translations/Messages_fr_FR.json (hunk 1 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/translations/Messages_fr_FR.json]: fixed.

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archives.constants.js (hunk 2 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archives.constants.js]: fixed.

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archives.constants.js (hunk 4 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archives.constants.js]: fixed.

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/onboarding/onboarding.controller.js (hunk 5 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/onboarding/onboarding.controller.js]: fixed.

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/translations/Messages_fr_FR.json [hunk 9]
```diff
@@ -4,12 +4,12 @@
...
-  "pci_projects_project_storages_cold_archives_guides_cold-archive-getting-started_description": "Apprendre à créer un bucket, uploader sa donnée, archiver, lister, désarchiver",
-  "pci_projects_project_storages_cold_archives_guides_billing_title": "Comprendre la facturation",
-  "pci_projects_project_storages_cold_archives_guides_billing_description": "Découvrez les règles de facturation",
-  "pci_projects_project_storages_cold_archives_guides_monitoring_title": "Monitorer ses archives",
-  "pci_projects_project_storages_cold_archives_guides_monitoring_description": "Comprendre les états des archives et les monitorer",
+  "pci_projects_project_storages_cold_archives_guides_overview_title": "Présentation du service Cold Archive",
+  "pci_projects_project_storages_cold_archives_guides_overview_description": "Découvrez le service, les concepts et la facturation",
+  "pci_projects_project_storages_cold_archives_guides_first-step_title": "Premier pas avec le service Cold Archive ",
+  "pci_projects_project_storages_cold_archives_guides_first-step_description": "Créez des conteneurs, archivez vos données, dégelez vos données, listez les metadonées...",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/translations/Messages_fr_FR.json:10]:
```suggestion
  "pci_projects_project_storages_cold_archives_guides_first-step_description": "Créez des conteneurs, archivez vos données, dégelez vos données, listez les métadonnée...",
```

Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/translations/Messages_fr_FR.json]: Done

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js (hunk 6 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js]: Done.

**Review by mohammed-zahaf — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Pull request merged**